keps/sig-node/4815-dra-partitionable-devices/README.md

```diff
@@ -267,12 +267,16 @@ The basic idea is the following:
    footprint of each device when a common set of attributes and capacities can
    be applied. The mixins themselves are introduced as a list of top-level
    objects directly next to the list of `Devices` inside a `ResourceSlice`.
-   They are not allocatable on their own.
+   They are not allocatable on their own. Entries in the `Includes` list can
+   reference mixins in any `ResourceSlice` in the same pool. References to
```

Comment:
@pohly I added this clarification as it seems logical to me and also helpful with large number of devices that might need to be split across multiple ResourceSlices. But it also means that we can't apply mixins to devices as part of conversion since there can be cross-ResourceSlice references. Do you have any opinion on whether we should allow this or keep it simpler by restricting it to just within ResourceSlice references?

Reply:
I prefer to keep it simpler by restricting it to just within ResourceSlice references.

That enables validation of mixin references (validation is per slice, not per pool) and a potential performance enhancement (expanding references once [when receiving a slice](https://github.com/pohly/kubernetes/commit/cb18e0f16094d7b669bfeb8207de48705dc046ce)).

Reply:
Yeah, agree that validation will be better if we restrict references to be within a single ResourceSlice. Also, it is easy to loosen this restriction later if we decide it is needed. I will update the language here.

Reply:
Updated the language to clarify that references can only be internal to a ResourceSlice.